src/utilities/countCompletedTodos.test.js

```diff
@@ -0,0 +1,32 @@
+import { countCompletedTodods } from './getNumberOfCompletedTodos';
+
+describe('The countCompletedTodos function', () => {
+  let todos;
+  describe('When provided with correct array of todos', () => {
```

Comment:
Personally, I'd start the "When" word with the small letter "w".

The countCompletedTodos function
when provided with the correct array of todos
should return the number of completed todos

but that might be a matter of preference.